resources/containers/view-content.tsx

```diff
@@ -1,25 +1,57 @@
 import React from "react";
 import styles from "./styles/view-content.module.css";
-import Markdown from "react-markdown";
 
 interface ViewContentProps {
-  content: string | string[];
-  type?: 'markdown'
+  content: object | string[];
 }
 
-const ViewContent: React.FC<ViewContentProps> = ({ content, type }) => {
+const renderObject = (obj: object) => {
+  console.log(obj);
+  return Object.entries(obj).map(([key, val]) => (
+    <div key={key}>
+      {key}:{" "}
+      {typeof val === "object" && val !== null
+        ? renderObject(val)
+        : String(val)}
+      <br />
+    </div>
+  ));
+};
+
+const ViewContent: React.FC<ViewContentProps> = ({ content }) => {
+  if (Array.isArray(content)) {
+    return (
+      <div className={styles.viewContent}>
+        {content.map((line, index) => (
+          <p key={index}>{line}</p>
+        ))}
+      </div>
+    );
+  }
+
+  console.log(typeof content);
 
-  if (type === 'markdown') {
-    const markdownContent = Array.isArray(content) ? content.join("\n") : content;
-    return (<Markdown>{markdownContent}</Markdown>)
+  if (typeof content === "object" && content !== null) {
```

Comment:
if type is object it cannot not be `null`, we can remove the 2nd condition which checks for `null`